lib/drivers/veres.js

```diff
@@ -636,13 +640,21 @@ api.info = async options => {
   async function looper(result) {
     // omit "not found" info if requested
     if(!options.retryShowFound || (options.retryShowFound && result.found)) {
-      _info(result, options);
+      if(callback) {
+        callback(null, result, options);
```

Comment:
We have found that calling a callback with multiple arguments is not a good idea.  So in this case, you'd return an object:
```
callback(err, {result, options});
```

Reply:
Ah, no prob, will fix. (I don't really need to return the options).